README.rst

```diff
@@ -34,7 +34,7 @@ See `doc/install.txt`_ in the source code archive for general information. Excep
 
 Python 3 or later is needed.
 
-The version in the pip repository is old. Instead, you can use pip to install the latest release from git: ``pip install git+https://github.com/linkchecker/linkchecker.git@v9.4.0``. See `#4 <https://github.com/linkchecker/linkchecker/pull/4>`_.
+The version in the pip repository is old. Instead, you can use pip to install the latest release from git: ``pip install git+https://github.com/linkchecker/linkchecker.git@v10.0.0rc1``. See `#4 <https://github.com/linkchecker/linkchecker/pull/4>`_.
```

Comment:
Let's make that `pip3` to avoid confusion (looks like the only correct option on Ubuntu 20.04 and works on Gentoo)